oscarapi/serializers/product.py

```diff
@@ -186,6 +186,7 @@ class Meta:
 
 
 class OptionSerializer(OscarHyperlinkedModelSerializer):
+    option_group = AttributeOptionGroupSerializer()
```

Comment:
Hi, I think this should be optional, the tests are failing atm!

Reply:
Also I think it is important to demonstrate the problem in a test, to make sure there are no regressions in the future